src/overlays/actors/ovl_Bg_Hidan_Firewall/z_bg_hidan_firewall.c

```diff
@@ -58,9 +57,9 @@ static ColliderCylinderInit sCylinderInit = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 1, 80, 100, MASS_IMMOVABLE };
 
-static UNK_PTR D_80886D04[] = {
-    0x06015D20, 0x06016120, 0x06016520, 0x06016920, 0x06016D20, 0x06017120, 0x06017520, 0x06017920,
-};
+static u64* D_80886D04[] = { gHidanObjectsFireball0Tex, gHidanObjectsFireball1Tex, gHidanObjectsFireball2Tex,
+                             gHidanObjectsFireball3Tex, gHidanObjectsFireball4Tex, gHidanObjectsFireball5Tex,
+                             gHidanObjectsFireball6Tex, gHidanObjectsFireball7Tex };
```

Comment:
```suggestion
                             gHidanObjectsFireball6Tex, gHidanObjectsFireball7Tex, };
```
If you put a comma here and then rerun format.sh it will look better.